app/org/sagebionetworks/bridge/validators/StudyValidator.java

```diff
@@ -66,16 +73,39 @@ public void validate(Object obj, Errors errors) {
             errors.rejectValue("identifier", "is required");
         } else {
             if (!study.getIdentifier().matches(BridgeConstants.BRIDGE_IDENTIFIER_PATTERN)) {
-                errors.rejectValue("identifier", "must contain only lower-case letters and/or numbers with optional dashes");
+                errors.rejectValue("identifier", BRIDGE_IDENTIFIER_ERROR);
             }
             if (study.getIdentifier().length() < 2) {
                 errors.rejectValue("identifier", "must be at least 2 characters");
             }
         }
         if (study.getActivityEventKeys().stream()
-                .anyMatch(k -> !k.matches(BridgeConstants.BRIDGE_IDENTIFIER_PATTERN))) {
-            errors.rejectValue("activityEventKeys", "must contain only lower-case letters and/or numbers with " +
-                    "optional dashes");
+                .anyMatch(k -> !k.matches(BridgeConstants.SYNAPSE_IDENTIFIER_PATTERN))) {
+            errors.rejectValue("activityEventKeys", SYNAPSE_IDENTIFIER_ERROR);
```

Comment:
Was this change intentional? Activity Event Keys have nothing to do with Synapse, so I'm not sure if the constants are correct.

Reply:
Yes. Your current study burst keys are camel-cased and there's no reason they can't be camel-cased, this validation seemed too stringent. I wanted it to be possible for the custom activity event keys and the automatic event keys to have the same format constraints. The synapse validation allows for upper-case letters and numbers and seemed more appropriate for this.